patroni/postgresql/available_parameters/0_postgres.yml

```diff
@@ -1336,6 +1426,9 @@ parameters:
   ssl_ecdh_curve:
   - type: String
     version_from: 90400
```

Comment:
ssl_ecdh_curve was removed (renamed) to ssl_groups, so it should have
version_till: 180000